caikit/runtime/model_management/loaded_model.py

```diff
@@ -118,7 +118,11 @@ def model(self) -> ModuleBase:
         self.wait()
         return self._model
 
-    def loaded(self) -> bool:
+    def loaded(self, require_instance: bool = False) -> bool:
```

Comment:
Why is the default `False`? I guess really, why make this an argument at all? It seems like the old logic was just wrong? Did you trace history to figure out why `or self._caikit_model_future.done()` was added?

Reply:
I didn't trace history but logically it seems `loaded` used to see if the `ModelFuture` was generally completed not just successfully completed. When I checked caikit general it doesn't seem like we use `loaded` outside the servicer so assumed it was used in a subclass/other implementation.

---
After looking through the history it seems like it is only used in the info servicer. I'm game to change this to the default